delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/routes/FilesRoutes.scala

```diff
@@ -290,6 +290,32 @@ final class FilesRoutes(
                     }
                   }
                 }
+              },
+              pathPrefix("register-update") {
+                (idSegment & indexingMode) { (id, mode) =>
+                  pathEndOrSingleSlash {
+                    operationName(s"$prefixSegment/files/{org}/{project}/register-update/{id}") {
```

Comment:
Not important but we don't need this `operationName` anymore as the Kamon feature is disabled because it made Prometheus crash by sending data with a lot of distinct values (which Prometheus is not fond about)